dtos/v2/account_dto_v2.py

```diff
@@ -0,0 +1,28 @@
+
+
+from dataclasses import dataclass
+import uuid
+
+from models.account_model import Account
+
+
+@dataclass
+class AccountLoginRequestV2(object):
+    email:str
+    password:str
+
+    def __init__(self, request: dict) -> None:
+        self.email = request["email"]
+        self.password = request["password"]
+        super().__setattr__('frozen', True)
+
+@dataclass
+class AccountLoginResponseV2(object):
```

Comment:
Should inherit from AbstractResponseV2 and implement a to_model method.

Reply:
converted. for response using from_model